config/config_test.go

```diff
@@ -986,9 +986,17 @@ var expectedConf = &Config{
 		},
 	},
 	TracingConfig: TracingConfig{
-		Endpoint:   "localhost:4317",
-		ClientType: TracingClientGRPC,
-		Insecure:   true,
+		Endpoint:    "localhost:4317",
+		ClientType:  TracingClientGRPC,
+		Insecure:    false,
+		Compression: "gzip",
+		Timeout:     model.Duration(5 * time.Second),
+		Headers:     map[string]string{"foo": "bar"},
```

Comment:
We should call `validateHeaders`on this one.

Reply:
I'd like to keep a consistent use of headers.

Reply:
Sure thing, I understand, I'll add this validation to the config unmarshalling logic